docs/content/getting-started/code-generation.md

```diff
@@ -178,8 +176,8 @@ We define __a schema__ and write our types as __data values__ in CUE.
 #### Schema & Data
 
 {{<codePane2
-	title1="schema.cue" file1="code/getting-started/code-generation/schema.html"
-	title2="data.cue" file2="code/getting-started/code-generation/data.html"
+  title1= "schema.cue" file1= "code/getting-started/code-generation/schema.html"
```

Comment:
what's up with the spaces between the = and the arg? Is that something your IDE did?